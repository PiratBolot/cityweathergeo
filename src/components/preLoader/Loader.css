.image_container_overlay {
    width: inherit;
    margin-left: auto;
    margin-right: auto;
    font-weight: bold;
}