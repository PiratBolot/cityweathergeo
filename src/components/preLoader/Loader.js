import React from 'react';
import './Loader.css';
import CircularSpinner from "./spinner/CircularSpinner";

class Loader extends React.Component {
    render() {
        return (
            <div className="image_container_overlay">
                <p className="label">Подождите, данные загружаются</p>
                <CircularSpinner/>
            </div>
        )
    }
}

export default Loader;