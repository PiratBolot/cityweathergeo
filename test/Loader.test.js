import React from 'react';
import { shallow } from '../enzyme/enzyme';
import { shallowToJson } from 'enzyme-to-json';

import Loader from "../src/components/preLoader/Loader";

describe('Pre-Loader', () => {
    it('should render correctly', () => {
        const output = shallow(
            <Loader />
        );
        expect(shallowToJson(output)).toMatchSnapshot();
    });
});