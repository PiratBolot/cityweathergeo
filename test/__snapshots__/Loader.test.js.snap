// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pre-Loader should render correctly 1`] = `
<div
  className="image_container_overlay"
>
  <p
    className="label"
  >
    Подождите, данные загружаются
  </p>
  <CircularSpinner />
</div>
`;
